settings.gradle

```diff
@@ -1 +1 @@
-rootProject.name = 'xm-uaa'
+rootProject.name = 'uaa'
```

Comment:
I think we should not change project name. Pls check if it was not mistake.

Reply:
fixed